tools/reviewer/job/sync/GithubClient.java

```diff
@@ -127,82 +132,106 @@ public IssueCommentsResponse getIssueComments(IssueCommentsRequest request) thro
     return builder.build();
   }
 
-  public void createIssueComment(CreateIssueCommentRequest request) {
+  public RepositoryCommentsResponse getRepositoryComments(RepositoryCommentsRequest request)
+      throws IOException {
+    RepositoryCommentsResponse.Builder builder = RepositoryCommentsResponse.newBuilder();
+    String response =
+        doRequest(
+                RequestMethod.GET,
+                String.format(GET_REPOSITORY_COMMENTS, request.getOwner(), request.getRepo()))
+            .replace("_links", "link")
```

Comment:
What if the comment itself has "_links" or "href"?
It's ok that the proto has `_links` and `href`. I wasn't aware it's a proto that's tied to the GitHub API.

Reply:
> What if the comment itself has "_links" or "href"?
Yes, it will be a problem.

Renamed to `_links` and `href`.